Bio/PDB/vectors.py

```diff
@@ -139,13 +141,14 @@ def refmat(p, q):
 
     Examples
     --------
+    >>> from Bio.PDB.vectors import refmat
     >>> p,q = Vector(1, 2, 3), Vector(2, 3, 5)
     >>> mirror = refmat(p, q)
     >>> qq = p.left_multiply(mirror)
     >>> print(q)
     <Vector 2.00, 3.00, 5.00>
     >>> print(qq)  # q and qq should be the same
-    <Vector 1.65, 3.30, 4.94>
+    <Vector 1.21, 1.82, 3.03>
```

Comment:
I see what you mean about the output being very different probably the test was always wrong...